src/transformers/pipelines/document_question_answering.py

```diff
@@ -0,0 +1,429 @@
+import re
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+
+from ..models.vision_encoder_decoder import VisionEncoderDecoderModel
```

Comment:
This is pytorch only so you cannot really do that.

In general pipelines prefer to stay agnostic to individual models.
We need a different way to differentiate.

Either something in the config is specific enough that we can differentiate (an option that exists in only one of the types of models and will always be defined on future models that work the same).

Or we can do `config.__class__.__name__ == "VisionEncoderDecoderConfig"`.

Using the config is better than the model since it will work for all 3 frameworks at once.

Reply:
I can do that!